bundles/org.openhab.binding.radiobrowser/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,142 @@
+				<label>Clicks</label>
+				<description>Send a click for every station you select. This helps to mark stations as popular and makes the
+					database more useful to other people.</description>
+				<default>true</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="languageCount" type="integer" min="1" required="true">
+				<label>Language Count</label>
+				<description>The minimum number of stations a language needs to have world wide before the binding lists it as an
+					option.</description>
+				<default>14</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="country">
+		<item-type>String</item-type>
+		<label>Country</label>
+		<description>Show only stations located in the selected country</description>
+	</channel-type>
+
+	<channel-type id="language">
+		<item-type>String</item-type>
+		<label>Language</label>
+		<description>Show only stations spoken in the selected language</description>
+	</channel-type>
+
+	<channel-type id="state">
+		<item-type>String</item-type>
+		<label>State</label>
+		<description>Show only stations located in the selected state, requires country to be selected first</description>
+	</channel-type>
+
+	<channel-type id="station">
+		<item-type>String</item-type>
+		<label>Station</label>
+		<description>A list of stations that you can select from to fetch the URI for</description>
+	</channel-type>
+
+	<channel-type id="genre">
+		<item-type>String</item-type>
+		<label>Genre</label>
+		<description>A list of genres or tags that you can select from</description>
+		<state readOnly="false">
+			<options>
+				<option value="ALL">Show All Genres</option>
+				<option value="christian">christian</option>
+				<option value="christmas">christmas</option>
+				<option value="classical">classical</option>
+				<option value="country">country</option>
+				<option value="dance">dance</option>
+				<option value="disco">disco</option>
+				<option value="edm">edm</option>
+				<option value="electronic">electronic</option>
+				<option value="folk">folk</option>
+				<option value="gospel">gospel</option>
+				<option value="hits">hits</option>
+				<option value="house">house</option>
+				<option value="indie">indie</option>
+				<option value="jazz">jazz</option>
+				<option value="lounge">lounge</option>
+				<option value="metal">metal</option>
+				<option value="music">music</option>
+				<option value="news">news</option>
+				<option value="oldies">oldies</option>
+				<option value="pop">pop</option>
+				<option value="rock">rock</option>
+				<option value="soul">soul</option>
+				<option value="sports">sports</option>
+				<option value="talk">talk</option>
+				<option value="techno">techno</option>
+				<option value="trance">trance</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="stream">
+		<item-type>String</item-type>
+		<label>Stream</label>
```

Comment:
```suggestion
		<label>Stream URI</label>
```

(does this make sense?)

Reply:
> NOOOOO. I do not want two word labels. The risk is that widgets break every time someone in the project decides that items should have no spaces Equipment_UpperCamelCase, then it gets moved to "-" spaced Equipment_Upper_Camel_Case and then to who knows what or if the change gets reversed.
> 
> Discussion on that we now have multiple widgets that are broken due to a core change.
> 
> https://community.openhab.org/t/mediaplayer-now-playing-and-control/126621/8
> 
> I have seen this madness go around in circles since I started with openhab 1.x and sadly it seems to not have stopped yet.

I'm sorry, but label rules are allready stable for a very long time. They should be 2-3 words and < 23 characters. The label as one word is perfectly fine, but it isn't a stream , its a uri.

Reply:
I strongly believe that stream is acceptable and does not confuse an end user. The URI points to a STREAM and only a STREAM so for me this makes sense. Otherwise you could say the same about the ICON channel, it holds a URI that points to an ICON.

Lastly I feel it is more consistent between textual users that see it in the UI and get confused. If the Label is the same as the actual channel I find that far less confusing as the channel is the same and you know that the naming convention never changes and is consistent :)

I do not care if it changes, I just feel strongly against 2 and more words for a binding that is going to rely heavily on a widget working out of the box.

Reply:
Ok. I understand your pov, lets leave it as is.

FTR: for an icon i have the same opinion :-)